ec2/spark_ec2.py

```diff
@@ -340,6 +346,27 @@ def is_active(instance):
     "t2.small":    "hvm",
 }
 
+# Return correct versions of Spark and Tachyon, given the supplied Spark version
+def get_spark_tachyon_version(opts):
+    spark_tachyon_map = {
```

Comment:
We have a map of valid Spark versions at the beginning of the file. Duplicating this here doesn't seem right.